.changeset/config.json

```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@2.3.1/schema.json",
+  "changelog": ["@changesets/changelog-github", { "repo": "refinedev/refine" }],
```

Comment:
is this `"repo": "refinedev/refine"` a typo? 

```suggestion
  "changelog": ["@changesets/changelog-github", { "repo": "refinedev/refine" }],
```

Reply:
yes, its typo. Thanks for catching it.

Reply:
fixed.